### PR TITLE
[Greenwich] Update link colour.

### PR DESCRIPTION
**Pull request by dracos**
Update to match their site. The front page colour didn't seem to do anything, and the visited seemed pointless. [skip changelog] Fixes https://github.com/mysociety/societyworks/issues/3435

### CODE REVIEW

File: web/cobrands/greenwich/base.scss [hunk 1]
```diff
@@ -137,7 +127,7 @@ form {
                         text-decoration: underline;
                     }
                     &:visited {
-                        color: $links_visited;
+                        color: darken($link-color, 10%);
```
Comment (lucascumsille) [inline web/cobrands/greenwich/base.scss:130]: I added this bit, it wasn't compiling the CSS because there was a remanent of `links_visited`

**Review by lucascumsille — COMMENTED** [review]
The other approach would be to remove the visited bit altogether.

File: web/cobrands/greenwich/base.scss (hunk 1 above)
Comment (dracos) [reply web/cobrands/greenwich/base.scss:130]: I don't think this ever happens at all, in fact - I can't see a fake-link within a #form_sign_in within a fieldset anywhere.

Comment (lucascumsille) [reply web/cobrands/greenwich/base.scss]: To be fair is not even needed. I just deleted in another commit

**Comment by lucascumsille** [comment]
@dracos I think this should be ready =)

File: web/cobrands/greenwich/base.scss (hunk 1 above)
Comment (dracos) [reply web/cobrands/greenwich/base.scss]: Sorry, I meant that entire block is never used, the whole thing. I've removed it.

**Pull request merged**